src/module-2/components/filters.css

```diff
@@ -0,0 +1,62 @@
+.filters{
+    width: 28.57vw;
+    height: 157.35vh;
+    position: relative;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    margin-left: 1vw;
+}
+.back-arrow, .filters-nav{
+    background-color: white;
+    box-shadow: 0 10px 20px rgba(0,0,0,0.25), 0 5px 5px rgba(0,0,0,0.22);  
+    border-radius: 8px;
+}
+.back-arrow {
+    height: 3.7vh;
+
+}
+.filters-nav {
+    width: 100%;
+    height: 140.83vh;
+    padding: 1.5vh 2.5vw;
+    font-size: 1.3rem;
+    font-weight: 500;
+    margin-top: 1.7vh;
+    
+}
+.filters-header, form[name="category"]{
+    display: flex;
+    flex-direction: row ;
+    justify-content: space-between;
+}
+.filters-header{
+    height: 4vh;
+    align-items: center;
+    font-size: 1.2rem;
+    font-weight: 700;
+    margin-bottom: 3.3vh;
+}
+
```

Comment:
no empty lines
or insert it before every selector to keep the same code style for the whole file